reactos/drivers/usb/usbhub/fdo.c

```diff
@@ -2044,7 +2106,6 @@ USBHUB_FdoHandlePnp(
         }
     }
 
-    Irp->IoStatus.Information = Information;
     Irp->IoStatus.Status = Status;
     IoCompleteRequest(Irp, IO_NO_INCREMENT);
```

Comment:
I've done this way to simplify reviewing process. After acceptance of logical part I'll refactor too.